transforms/helpers/transform.ts

```diff
@@ -97,8 +97,16 @@ function _maybeTransformEmberObjects(
     unobserves: false,
   };
 
+  const eoExtendExpressionPaths = getEOExtendExpressionCollection(j, root);
+
+  if (eoExtendExpressionPaths.length === 0) {
+    logger.warn(
```

Comment:
Does this make sense as a warning? What conditions cause it to happen?

Reply:
I put it in during my debugging because switching to the babel parser initially broke the codemod's detection of EmberObject extend expressions, which took some time to figure out (the symptom was just that nothing was getting transformed). I decided to leave the logging in in case things break again in the future. I chose `warn` to follow the conventions already in the project for handling skipped files in the logging: 

https://github.com/ember-codemods/ember-native-class-codemod/blob/96fa9bf0dd99be5281a9e245d688bd643b1033e9/transforms/helpers/parse-helper.js#L179-L194